main.go

```diff
@@ -48,7 +48,11 @@ func generateFake(interfaceName, sourcePackageDir, importPath, outputPath, desti
 	var err error
 	var iface *model.InterfaceToFake
 	if sourcePackageDir == "" {
-		iface, err = locator.GetInterfaceFromImportPath(interfaceName, importPath)
+		var vendorPaths []string
+		vendorPaths, err = locator.VendorPathsForDirPath(".")
```

Comment:
I agree, but I wasn't sure how to do that.

I would rename `GetInterfaceFromImportPath` to an unexported function, change all calls to that, then create another `GetInterfaceFromImportPath` without the last parameter, and call `locator.VendorPathsForDirPath(".")` before calling the unexported function.

There's also another PR that changes these functions, so that complicates things a bit.